modules/siddhi-extensions/map/src/test/java/org/wso2/siddhi/extension/map/RemoveFunctionExtensionTestCase.java

```diff
@@ -96,4 +97,26 @@ public void receive(Event[] events) {
         Assert.assertTrue(eventArrived);
         executionPlanRuntime.shutdown();
     }
+
+    @Test(expected = ExecutionPlanValidationException.class)
+    public void testRemoveFunctionExtension1() throws InterruptedException {
+        log.info("RemoveFunctionExtension TestCase with test attributeExpressionExecutors length");
+        SiddhiManager siddhiManager = new SiddhiManager();
+
+        String inStreamDefinition = "\ndefine stream inputStream (symbol string, price long, volume long);";
+        String query = ("@info(name = 'query1') from inputStream select symbol,price, "
+                + "map:create() as tmpMap insert into tmpStream;"
+                + "@info(name = 'query2') from tmpStream  select symbol,price,tmpMap, map:put(tmpMap,symbol,price) as map1"
+                + " insert into outputStream;"
+                + "@info(name = 'query3') from outputStream  select map1, map:remove(map1) as map2"
+                + " insert into outputStream2;"
+        );
+
+        ExecutionPlanRuntime executionPlanRuntime = siddhiManager.createExecutionPlanRuntime(inStreamDefinition + query);
+
+
+        InputHandler inputHandler = executionPlanRuntime.getInputHandler("inputStream");
```

Comment:
Below three lines are not needed. createExecutionplanRuntime will throw the exception